config/initializers/cypress_rails.rb

```diff
@@ -0,0 +1,19 @@
+return unless Rails.env.test?
```

Comment:
This file is generated by the cypress-rails gem when you initialize it. The only tweak I made was to seed data in the `before_server_start` hook.